src/sentry/snuba/metrics/datasource.py

```diff
@@ -1,5 +1,7 @@
 from __future__ import annotations
 
+from sentry.sentry_metrics.visibility import get_blocked_metrics
```

Comment:
nit: do we plan on putting anything else in this module? If not, then i would call it `blocking` or `metrics_blocking` or would flatten it out by butting exports and errors in `metrics_blocking.py`

Reply:
Yes we are, all the deletion logic will go there. It's called visibility exactly because it's a generic term that both have in common.